packages/deployed-backend-client/src/backend_output_client_factory.ts

```diff
@@ -11,6 +11,8 @@ export enum BackendOutputClientErrorType {
   NO_OUTPUTS_FOUND = 'NoOutputsFound',
   DEPLOYMENT_IN_PROGRESS = 'DeploymentInProgress',
   NO_STACK_FOUND = 'NoStackFound',
+  EXPIRED_TOKEN = 'ExpiredToken',
+  ACCESS_DENIED = 'AccessDenied',
```

Comment:
I was conflicted if we should have a more generic `CREDENTIAL_ERROR` key here to cover both expired token and access denied errors but went with more specificity.

Reply:
I think `CREDENTIAL_ERROR` is fine. (expired token, missing token, invalid key, etc are credentials errors).
But, `ACCESS_DENIED` is not subset of `CREDENTIAL_ERROR` .